api/DOMTokenList.json

```diff
@@ -93,6 +93,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "add_multiple_arguments": {
+          "__compat": {
+            "description": "Multiple arguments for <code>add()</code>",
+            "support": {
+              "chrome": {
+                "version_added": "24"
+              },
+              "chrome_android": {
+                "version_added": true
```

Comment:
This should be "25". 

The first version of Chrome Android was 18. The second was 25. Anything that landed between those lists whichever follows as the first version.